src/app/Models/Candidate.ts

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
![image](https://user-images.githubusercontent.com/1723469/198592015-04d810e1-9e63-4d74-bd91-00cde7f245ec.png)

folder and file structure must look like this..